back/spec/policies/initiative_policy_spec.rb

```diff
@@ -256,21 +320,40 @@
         it { is_expected.not_to permit(:create) }
         it { is_expected.not_to permit(:update) }
         it { is_expected.not_to permit(:destroy) }
+        it { is_expected.not_to permit(:accept_invite) }
 
         it 'does not index the initiative' do
           expect(scope.resolve.size).to eq 1
         end
       end
 
-      # Author can see initiative, but not edit, if review feature is active.
+      # Author can see initiative, but not edit, if review feature is active,
+      # and when status in InitiativeStatus::NOT_REVIEW_CODES
       context 'for a user who is author of the initiative' do
         let(:user) { author }
 
         it { is_expected.to permit(:show) }
         it { is_expected.to permit(:by_slug) }
         it { is_expected.to permit(:create) }
-        it { is_expected.to permit(:update) }
+        it { is_expected.to permit(:update) } # TODO: This is confusing, as editing_locked should be true when status proposed!
         it { is_expected.to permit(:destroy) }
+        it { is_expected.not_to permit(:accept_invite) }
+
+        it 'indexes the initiative' do
+          expect(scope.resolve.size).to eq 1
+        end
+      end
+
+      context 'for a user who is cosponsor of the initiative' do
+        let(:user) { create(:user) }
```

Comment:
This file and in general our policy specs have so many examples `it { is_expected.to permit(:...) }`. I'm wondering how fast they are and how we can speed them up. Because now `back-test` job is the slowest one. Maybe simply using `let_it_be` instead of `let` could help significantly.

Definitely not a blocker.

Reply:
I agree. I'd like to pair with you at some point to see if we can reduce the n of tests. I suspect we can radically do so.

Reply:
> we can reduce the n of tests

Maybe it's not needed if every test just checks the policy without creating any DB records (using `let_it_be`).